app/src/main/java/io/gnosis/safe/utils/AppUtils.kt

```diff
@@ -84,3 +87,13 @@ fun showConfirmDialog(
         title = if (title == null) null else context.resources.getString(title)
     ).show()
 }
+
+fun String.safeParseColorWithDefault(context: Context, defaultColor: Int, tracker: Tracker? = null): Int {
```

Comment:
String.toColor(...  would be a better name
defaultColor -> failsafeColor
then everything should be clear and we don't need such a long name

Reply:
defaultColor should have a default value

Reply:
also I think we don't need tracker here. Timber.e should be enough as all errors are submitted as non-fatals to firebase

Reply:
I also moved it to StringUtils